### DISCUSSION TITLE
AppImage update option

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#829] [Ideas]
### What feature would you like to see?

I would like to be able to update Zen installed as an AppImage without checking on the website. I know some AppImage apps have this option.

![image](https://github.com/user-attachments/assets/23ea4817-a7fd-41fc-8394-0f38e516579d)

It says that "Updates disabled by your organization"

And GearLevel doesn't notify me of new versions for some reason

![image](https://github.com/user-attachments/assets/878c37a6-2586-4b23-bf73-b4cdf96a7fc2)

### What platform are you seeing the problem on?

Linux

### Additional context

`Linux stupid-dragon 6.10.6-arch1-1 #1 SMP PREEMPT_DYNAMIC Mon, 19 Aug 2024 17:02:39 +0000 x86_64 GNU/Linux`
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/466_

**Comment by SomeDudeBrosky** [comment]
Use the flatpak.

**Reply by eboye** [reply]
Well, if AppImage exist and someone want to use it, then it's up to them.

I use flatpak version on my Fedora machine and AppImage on my Arch machine.

Flatpak version does some drawbacks like downloading things in flatpak temp folder which is not always great if you want to open them directly in other apps.

**Reply by Guchdorg** [reply]
> Use the flatpak.

Fair enough about the flatpak comment.  There still should be an option to toggle notification when a new update is available.
